src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+
+        System.out.println("Bulldozer is started work");
+
```

Comment:
and here too)

make the same with all methods

Reply:
done